core/domain/learner_goals_services.py

```diff
@@ -0,0 +1,143 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Services for the learner goals feature of the learner dashboard."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import user_domain
+from core.platform import models
+import feconf
+
+(user_models,) = models.Registry.import_models([models.NAMES.user])
+
+MAX_CURRENT_GOALS_COUNT = feconf.MAX_CURRENT_GOALS_COUNT
```

Comment:
Why this initialization? Can't you just access `feconf.MAX_CURRENT_GOALS_COUNT` where required?

Reply:
Done.